sinkingship.py

```diff
@@ -135,6 +143,17 @@ def _download_asset(post_dir, url, suffix=""):
         print(f"Error trying to download URL {url!r} ({err})")
         return
 
+def _download_text_src(post_dir, body):
+    urls = re.findall("src=\"(https?://\S+)\"", body)
```

Comment:
Are you trying to scrape all the links from a page? Or something else? The posts I've saved already had the content included.

(Also, I prefer to use BeautifulSoup for HTML parsing, FYI.)